### PR TITLE
Added: dataarts section in into catalog.yaml

### PR DESCRIPTION
**Pull request by mmazec**

### CODE REVIEW

**Comment by mmazec** [comment]
Hi @vladimirvshivkov please can you merge this request. I've seen that "dataarts" proxy is already in the new "python-otcextensions" release.

**Review by vladimirvshivkov — APPROVED** [review]

File: config.yaml [hunk 1]
```diff
@@ -108,6 +108,7 @@ matrix:
           - css
           - cts_skip_unver
           - cts
+          - dataarts
```
Comment (vladimirhasko) [inline config.yaml:111]: @mmazec you want to enable this metric to be monitored only from eu-de to eu-de
or also from eu-nl to eu-de?

File: mp-prod/conf.d/health_metrics.yaml [hunk 2]
```diff
@@ -390,6 +390,20 @@ health_metrics:
         weight: 1
       - expression: "css.api_down"
         weight: 2
+  ### DATAARTS
+  dataarts:
+    service: dataarts
+    component_name: "Data Arts Studio"
```
Comment (vladimirhasko) [inline mp-prod/conf.d/health_metrics.yaml:396]: please fix it to be either Data Arts Studio or Data Arts Service
but not 2 different names (health metric component name and service name in sdb_prod/catalog.yaml

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (mmazec) [reply config.yaml:111]: @vladimirhasko added monitoring zone from EU-NL.

File: mp-prod/conf.d/health_metrics.yaml (hunk 2 above)
Comment (mmazec) [reply mp-prod/conf.d/health_metrics.yaml]: Fixed naming of DataArts.

**Pull request merged**